src/opcodes.rs

```diff
@@ -4,6 +4,7 @@ pub struct Opcode {
     pub pops: usize,
     pub pushes: usize,
     pub sign: Option<&'static str>,
+    pub alt: &'static str,
```

Comment:
maybe we use a more descriptive term for "alt"

Reply:
Any ideas? Maybe something related to "return value" since it adds that to the stack rather than the entire function call

Reply:
Renamed to stack output and show stack output (in line with evm.codes terminology) - wdyt @shafu0x